tests/PortingAssistant.Client.UnitTests/TelemetryClientFactoryTest.cs

```diff
@@ -25,5 +26,41 @@ public void EnablingDefaultCredentials_CreatesTelemetryClient()
 
             Assert.IsTrue(isClientCreated);
         }
+
+        [Test]
+        public void CreatesTelemetryClient_WithCredentials()
+        {
+            string profile = null;
+            var telemetryConfig = new TelemetryConfiguration()
+            {
+                InvokeUrl = @"https://8cvsix1u33.execute-api.us-east-1.amazonaws.com/gamma",
+                Region = "us-east-1",
+            };
+            AWSCredentials credentials = new BasicAWSCredentials("accessKey", "secretKey");
+            ITelemetryClient client;
+
+            bool result = TelemetryClientFactory.TryGetClient(profile, telemetryConfig, out client, awsCredentials: credentials);
+
+            Assert.IsTrue(result);
+            Assert.IsNotNull(client);
+        }
+
+        [Test]
+        public void CreatesTelemetryClient_WithoutProfileOrCredentials_Failed()
+        {
+            string profile = null;
+            var telemetryConfig = new TelemetryConfiguration()
+            {
+                InvokeUrl = @"https://8cvsix1u33.execute-api.us-east-1.amazonaws.com/gamma",
```

Comment:
which endpoint is this, the pasc one?

Reply:
no this is the old endpoint following other unit tests. Let me try to update it to a dummy.